### ISSUE TITLE
[BUG]: LLVM Opt Pipeline viewer seems to be broken

### ISSUE THREAD

**Issue by tlemo** [compiler-explorer/compiler-explorer#4176]
### Describe the bug

The LLVM opt pipeline only shows one pass

### Steps to reproduce

https://godbolt.org/z/K5Thsfv3e

### Expected behavior

LLVM passes show up in the pass list

### Reproduction link

https://godbolt.org/z/K5Thsfv3e

### Screenshots

![image](https://user-images.githubusercontent.com/13824553/197099224-9361906f-50d0-4d31-b51b-066ce2adf192.png)

**Comment by jeremy-rifkin** [comment]
It is indeed broken, or was :) This was reported the other day by #4169 and fixed in #4170. Just waiting on a site deploy.

**Comment by tlemo** [comment]
Thanks @jeremy-rifkin. Closing this one as duplicate of #4169
